src/spikeinterface/benchmark/benchmark_plot_tools.py

```diff
@@ -314,6 +319,38 @@ def plot_performances_comparison(
     return fig
 
 
+def plot_performances_losses(study, case0, case1, metric=["accuracy"], figsize=None):
```

Comment:
```suggestion
def plot_performance_losses(study, case0, case1, metric=["accuracy"], figsize=None):
```
The plural should be only in losses